queue/abstract/queue_session.lua

```diff
@@ -210,7 +223,9 @@ local method = {
     disconnect = disconnect,
     grant = grant,
     on_session_remove = on_session_remove,
-    start = start
+    start = start,
```

Comment:
Add a description for methods that have become public.

Reply:
Description added